examples/hello-world-client/hello_world_client.bal

```diff
@@ -3,7 +3,7 @@ import ballerina/io;
 
 public function main() {
     // Create an HTTP client to interact with a remote endpoint.
-    http:Client clientEP = new("http://www.mocky.io");
+    http:Client clientEP = new ("http://www.mocky.io");
```

Comment:
Doesn't a space here look odd? This is basically like a function call, and we don't use spaces between the function name and the parenthesis

Reply:
It looks odd to me too. 

The confusion I guess is with this vs specifying the type explicitly, where the arguments kinda look like they are bound to the type. 

```ballerina
http:Client clientEP = new http:Client("http://www.mocky.io");
```

Reply:
It seems the rule here is keywords are separated by a space always according to the style guide. IDE formatting is also done according to that. WDYT @NipunaMarcus?

Reply:
@anupama-pathirage - Yes, keywords should be separated by single space .. especially in this scenario as it's not a function invocation... there seems to be a lack of information regarding this in the style guide... I will update the document to reflect this clearly ...